metpy/calc/tools.py

```diff
@@ -81,56 +81,33 @@ def find_intersections(x, a, b):
 
     Returns
     -------
-        An array of (x,y) intersections of the lines.
+        A tuple (x, y) of array-like with the x and y coordinates of the
+        intersections of the lines.
     """
 
     # Find the index of the points just before the intersection(s)
     nearest_idx = nearest_intersection_idx(a, b)
+    next_idx = nearest_idx + 1
 
-    # Make an empty array to hold what we'll need for calculating
-    # approximate intersections. Each row is an intersection
+    # x-values around each intersection
+    x0 = x[nearest_idx]
+    x1 = x[next_idx]
 
-    # x1, x2, ya1, ya2, yb1, yb2
-    line_data = np.zeros((np.size(nearest_idx), 6))
+    # y-values around each intersection for the first line
+    a0 = a[nearest_idx]
+    a1 = a[next_idx]
 
-    # m1, m2, b1, b2
-    line_fits = np.zeros((np.size(nearest_idx), 4))
+    # y-values around each intersection for the second line
+    b0 = b[nearest_idx]
+    b1 = b[next_idx]
 
-    # xi, yi
-    intersections = np.zeros((np.size(nearest_idx), 2))
+    # Calculate the x-intersection of the lines: (b2 - b1) / (m1 - m2)
```

Comment:
I think typesetting might be a bit much in this case, since it's not exactly a fundamental formula. I've updated the comments and just phrased it as running through some algebra, since that's what I was able to just do by hand,